Author: Daniel Baumann <daniel@debian.org>
Description:
 Makes rdiff aware of the -i and -z shortcuts for --gzip and --bzip2 options
 (Closes: #435894).

diff -Naurp librsync.orig/rdiff.c librsync/rdiff.c
--- librsync.orig/rdiff.c	2008-07-01 15:29:15.000000000 +0200
+++ librsync/rdiff.c	2008-07-01 15:25:17.000000000 +0200
@@ -97,8 +97,8 @@ const struct poptOption opts[] = {
     { "sum-size",    'S', POPT_ARG_INT,  &strong_len },
     { "statistics",  's', POPT_ARG_NONE, &show_stats },
     { "stats",        0,  POPT_ARG_NONE, &show_stats },
-    { "gzip",         0,  POPT_ARG_NONE, 0,             OPT_GZIP },
-    { "bzip2",        0,  POPT_ARG_NONE, 0,             OPT_BZIP2 },
+    { "gzip",        'z', POPT_ARG_NONE, 0,             OPT_GZIP },
+    { "bzip2",       'i', POPT_ARG_NONE, 0,             OPT_BZIP2 },
     { "paranoia",     0,  POPT_ARG_NONE, &rs_roll_paranoia },
     { 0 }
 };
